cli/repair/advertise_uri.go

```diff
@@ -0,0 +1,25 @@
+package repair
+
+import (
+	"fmt"
+
+	"github.com/tarantool/cartridge-cli/cli/context"
+)
+
+func patchConfAdvertiseURI(workDir string, ctx *context.Ctx) ([]string, error) {
```

Comment:
Is it really needed?
This function just call another function.

Reply:
Yes, patch functions call `patchConf` wrapper and can be used by `repair.Run` function.

Look at `patchConfRemoveInstance`.